nuspec/chocolatey/ChocolateyGUI.nuspec

```diff
@@ -24,22 +24,15 @@ Chocolatey GUI is a nice GUI on top of the Chocolatey command line tool.
 * See detailed **package information**
 
 ## Notes
-This package will only work correctly on Windows 7 SP1 through Windows 10 (1708) or Windows Server 2008 R2 SP1 through Windows Server 2016, and requires .NET Framework 4.5.2 at minimum. 
+This package will only work correctly on Windows 7 SP1 through Windows 10 (1708) or Windows Server 2008 R2 SP1 through Windows Server 2016, and requires .NET Framework 4.5.2 at minimum.
   </description>
     <summary>A GUI for Chocolatey</summary>
   <releaseNotes>
 All release notes for Chocolatey GUI can be found on the GitHub site - https://github.com/chocolatey/ChocolateyGUI/releases
   </releaseNotes>
     <tags>chocolateygui chocolatey admin foss</tags>
     <dependencies>
-      <dependency id="Chocolatey" version="[0.10.3, 0.11)" />
+      <dependency id="chocolateygui.install" />
```

Comment:
I don't know if you remember the talk with Rob about using explicit version as a dependency a few months back, and why it was decided it should be used?

But basically, because then the user would be getting the version they specified to install on the meta package.

Isn't this possible to do with the following line?
```xml
<dependency id="chocolateygui.install" version="[$version$]" />
```
I've never tested this though, so I don't know if it works or not.

Reply:
Are you reading my todo list somehow?!?!  I had the same thought.  Have a note to check how this can best be done. 😄